components/header/Header.vue

```diff
@@ -165,7 +165,7 @@ const links = [
   {
     title: 'resources',
     displayTitle: 'Resources',
-    href: '/resources'
+    href: '/resources?type=sparcPartners'
```

Comment:
Nit: we should probably reference the resource id from contentful for the resources link.